### PR TITLE
Change config from toml to yaml

### PR DESCRIPTION
**Pull request by wuminzhe**

### CODE REVIEW

**Comment by xiaoch05** [comment]
I recommend that we separate the configures of d2e and e2d in the configure file if some item is used by only one of them.

**Comment by wuminzhe** [comment]
@xiaoch05 I did this before like what you said, but after discussion, they were moved to ethereum and darwinia section.

File: src/service/ethereum.rs [hunk 1]
```diff
@@ -284,34 +284,41 @@ impl EthereumService {
...
 		let filters = [
-			&config.eth.contract.bank,
-			&config.eth.contract.issuing,
-			&config.eth.contract.relay,
+			&config.ethereum.contract.bank,
+			&config.ethereum.contract.issuing,
+			&config.ethereum.contract.relay,
 		]
 		.iter()
 		.map(|c| {
```
Comment (wuminzhe) [inline src/service/ethereum.rs:309]: the code changed here(310~322) need to be reviewed carefully

**Review by xiaoch05 — APPROVED** [review]

File: .maintain/config/mainnet.sample.yml [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+  # (optional) the person who will ecdsa sign new mmr root or new authority set
+  # Disable by removing or commenting
+  authority:
+    # private key to sign ecdsa messages
+    # the signature will be submitted to Darwinia by darwinia.relayer
+    private_key: 0x... 
+
+darwinia:
+  # darwinia node websocket json-rpc endpoint
+  rpc: wss://crab.darwinia.network
```
Comment (HackFisher) [inline .maintain/config/mainnet.sample.yml:49]: Need to update the rpc to correct one, not crab

**Review by HackFisher — CHANGES_REQUESTED** [review]

**Review by HackFisher — APPROVED** [review]

**Pull request merged**